packages/api/src/controllers/room.ts

```diff
@@ -146,33 +152,43 @@ app.post(
       }
     );
     console.log("egress started", info);
-    await db.room.update(room.id, {
-      egressId: info.egressId,
-      updatedAt: Date.now(),
-    });
     res.status(204).end();
   }
 );
 
 app.delete("/:roomId/egress", authorizer({}), async (req, res) => {
-  const room = await getRoom(req);
-
-  if (room.egressId === undefined || room.egressId == "") {
-    throw new BadRequestError("egress has not started");
-  }
+  await getRoom(req);
 
   const egressClient = new EgressClient(
     req.config.livekitHost,
     req.config.livekitApiKey,
     req.config.livekitSecret
   );
+  const currentEgress = await egressClient.listEgress(req.params.roomId);
+
+  let found = false;
+  for (const egress of currentEgress) {
+    if (isEgressRunning(egress)) {
+      found = true;
+      const info = await egressClient.stopEgress(egress.egressId);
+      console.log("egress stopped", info);
```

Comment:
Can you make these logs more structured e.g. add `roomId=${}`, `egressId=`, etc. Might still want to log the full info in the end anyway (can double-stringify it so it's parsable by Loki, tho the full JSON is prolly more useful for us to copy and inspect it individually anyway)